Automattic/wc-calypso-bridge
# Hide installed extensions from search results on the "Extensions" screen

When searching for extensions via the "Extensions" screen, we currently show installed extensions as part of the search results. This can lead to confusion and potentially trying to purchase an extension twice.

While we'll see about getting this into WooCommerce core as well, this pull request replaces the "Extensions" screen with an extended class, containing additional logic to hide installed extensions from the search results.

### Testing Instructions
- Ensure you have at least one WooCommerce extension installed and activated on your test store (I've been using WooCommerce Brands as a test case).
- Visit "WooCommerce > Extensions" in WP Admin.
- Search for a keyword contained in the name of the extension above.
- The extension installed above should not show in the search results.

### Screenshots

![image](https://user-images.githubusercontent.com/880803/49286575-3a507c80-f4a3-11e8-9e47-3a1081205e77.png)

cc @timmyc @sararosso
